ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/platform.j2

```diff
@@ -35,6 +35,18 @@
 | {{ sandsetting }} | {{ platform.sand[sandsetting] }} |
 {%                 endif %}
 {%             endfor %}
+{%             if platform.sand.qos_maps is arista.avd.defined %}
+
+#### QOS Mapping
```

Comment:
Please change this to `Internal Network QoS Mapping`. This is an internal qos construct only relevant on J2 platforms.

Reply:
Sure